repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -16,21 +16,25 @@
     'rhel8-crb': 'codeready-builder-for-rhel-8-x86_64-rpms',
     'rhel8-supplementary': 'rhel-8-for-x86_64-supplementary-rpms'}
 
-Event = namedtuple('Event', ['action',   # A string representing an event type (see EVENT_TYPES)
-                             'in_pkgs',  # A dictionary with packages in format {<pkg_name>: <repository>}
-                             'out_pkgs'  # A dictionary with packages in format {<pkg_name>: <repository>}
+Event = namedtuple('Event', ['action',        # A string representing an event type (see EVENT_TYPES)
+                             'in_pkgs',       # A dictionary with packages in format {<pkg_name>: <repository>}
+                             'out_pkgs',      # A dictionary with packages in format {<pkg_name>: <repository>}
+                             'architectures'  # A list of strings representing architectures
                              ])
 
 EVENT_TYPES = ('Present', 'Removed', 'Deprecated', 'Replaced', 'Split', 'Merged', 'Moved', 'Renamed')
+ARCHITECTURES = ['x86_64', 'aarch64', 'ppc64le', 's390x']
```

Comment:
To avoid duplicate code, you can use [this](https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/el7toel8/libraries/config/architecture.py#L11).